src/crypto-api.ts

```diff
@@ -185,6 +185,17 @@ export interface CryptoApi {
      * @returns True if secret storage is ready to be used on this device
      */
     isSecretStorageReady(): Promise<boolean>;
+
+    /**
+     * Get the status of our cross-signing keys.
+     *
+     * - if there is a public key on this device
+     * - if the private master, userSigning and selfSigning keys are store in the SecretStorage
+     * - if the private master, userSigning and selfSigning keys are cached locally
```

Comment:
This feels a bit redundant? It's just repeating what's in `CrossSigningStatus`. Maybe we can instead include a summary in the `@returns` tag...